tests/specs/graph/jsr/FastCheck_Overloads.txt

```diff
@@ -0,0 +1,89 @@
+# https://jsr.io/@scope/a/meta.json
+{"versions": { "1.0.0": {} } }
+
+# https://jsr.io/@scope/a/1.0.0_meta.json
+{ "exports": { ".": "./mod.ts" } }
+
+# https://jsr.io/@scope/a/1.0.0/mod.ts
+export class argsToArray {
+  argsToArray(..._args: string[]): void;
+  argsToArray(..._args: number[]): void;
+  argsToArray(..._args: unknown[]): void {}
+}
+
+function overloadSpread(...args: string[]);
+function overloadSpread(...args: number[]);
+export function overloadSpread(...args: unknown[]) {
+  return args;
+}
+
+# mod.ts
+import 'jsr:@scope/a'
+
+# output
+{
+  "roots": [
+    "file:///mod.ts"
+  ],
+  "modules": [
+    {
+      "kind": "esm",
+      "dependencies": [
+        {
+          "specifier": "jsr:@scope/a",
+          "code": {
+            "specifier": "jsr:@scope/a",
+            "span": {
+              "start": {
+                "line": 0,
+                "character": 7
+              },
+              "end": {
+                "line": 0,
+                "character": 21
+              }
+            }
+          }
+        }
+      ],
+      "size": 22,
+      "mediaType": "TypeScript",
+      "specifier": "file:///mod.ts"
+    },
+    {
+      "kind": "esm",
+      "size": 314,
+      "mediaType": "TypeScript",
+      "specifier": "https://jsr.io/@scope/a/1.0.0/mod.ts"
+    }
+  ],
+  "redirects": {
+    "jsr:@scope/a": "https://jsr.io/@scope/a/1.0.0/mod.ts"
+  },
+  "packages": {
+    "@scope/a": "@scope/a@1.0.0"
+  }
+}
+
+Fast check https://jsr.io/@scope/a/1.0.0/mod.ts:
+  {}
+  export class argsToArray {
+    argsToArray(..._args: string[]): void;
+    argsToArray(..._args: number[]): void;
+    argsToArray(...param0?: any): any {
+      return {} as never;
+    }
+  }
+  function overloadSpread(...args: string[]);
+  function overloadSpread(...args: number[]);
+  export function overloadSpread(...param0?: any): any {
+    return {} as never;
+  }
+  --- DTS ---
+  export declare class argsToArray {
+    argsToArray(..._args: string[]): void;
+    argsToArray(..._args: number[]): void;
+  }
+  declare function overloadSpread(...args: string[]);
+  declare function overloadSpread(...args: number[]);
+  export declare function overloadSpread(...param0?: any): any;
```

Comment:
The implementation is not included in the dts output from `tsc` as it cannot be referenced. Might have to do with the overloads not being exported.

Reply:
yup that was it, fixed.